docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -10,6 +13,10 @@ independently.
 However, there may be cases where several policies conflict. Those conflicts are notified to the user through the
 `ReturnCodes` that the QoS setter functions returns.
 
+Each Qos Policy has a unique ID defined in the |QosPolicyId_t-api| enumerator.
+This ID is used in some :ref:`dds_layer_core_status` instances to identify the specific Qos Policy
+to which the Status refers.
```

Comment:
I feel that there is no explanation of what a Policy and status are and how they are related. Of course it'd fit more on core introduction or even on DDS layer's, but still

Reply:
The concepts are explained to some degree on the `Status` and `Policy` intros:
```
Each Entity is associated with a set of Status objects whose values represent the communication status of that Entity. 
```
```
The Quality of Service (QoS) is used to specify the behavior of the Service, allowing the user to define how each entity will behave. To increase the flexibility of the system, the QoS is decomposed in several QoS Policies that can be configured independently. 
```

There is no relationship between them, except that the `offered_incompatible_qos` and `requested_incompatible_qos` status keep a record of incompatible policies between readers and writers.

But I concede that the sentence here about some Status using the Policy ID may be misleading. Probably we can remove it altogether, avoid the confusion. The use of the ID is clearly described on the aforementioned Status.

Reply:
Sorry, I don't think any of these explanations are unclear, it's more that I need to go all the way to the status section to get a basic definition of what a status is. I feel that it could come earlier on, so when I go to the status section it's because I know what a status is, if that makes sense. We could simply copy those small sentences in the `core` intro somewhere, I think that'd be enough.